src/Chainweb/Pact/PactService/Checkpointer/Internal.hs

```diff
@@ -493,7 +493,9 @@ getBlockHistory db blockHeader d = do
             [SInt s,SInt e]
             [RInt,RText,RBlob]
       forM r $ \case
-        [SInt txid, SText key, SBlob value] -> (key,fromIntegral txid,) <$> Pact5.toTxLog (Pact5.renderDomain d) key value
+        [SInt txid, SText key, SBlob value] ->
+            (key,fromIntegral txid,)
+            <$> Pact5.toTxLog (_chainwebVersion blockHeader) (_chainId blockHeader) (view blockHeight blockHeader) (Pact5.renderDomain d) key value
```

Comment:
I know this (and the other diffs in this file) is a bit of an unused area of the code but it looks fine.